langserver/workspace_refs.go

```diff
@@ -33,8 +36,23 @@ func (h *LangHandler) handleWorkspaceReference(ctx context.Context, conn JSONRPC
 		pkgPat = h.init.RootImportPath + "/..."
 	}
 
+	var paralellism int
+	e := os.Getenv("WORKSPACE_REFERENCE_PARALLELISM")
```

Comment:
Yeah.. I actually think it is a bad idea to use environment variables to tweak things like this anyway. If you think about it, the person modifying the environment has to choose between slow and fast with no good in-between. In an ideal world, work is scheduled based on what work the language server itself is doing and what global resources are allocated to it as a whole. This is harder to implement, so.. e95b4c79df487d9aa01c4bcfe5a4f6047840c423 as you suggest for now.